rotkehlchen/api/v1/fields.py

```diff
@@ -560,7 +560,7 @@ def _deserialize(
             **_kwargs: Any,
     ) -> SupportedBlockchain:
         try:
-            chain = SupportedBlockchain.deserialize_from_name(value)
+            chain = SupportedBlockchain.deserialize(value)
```

Comment:
the one place where my change must have fucked things up is this one. And probably this is not needed. But the function and especially settings.py 333 is absolutely needed.

This duality is annoying but it's due to upgrading from evm chains to something more generic.